documentation/bolt.ditamap

```diff
@@ -86,6 +86,7 @@
         <topicref href="applying_manifest_blocks.md" format="markdown"/>
         <topicref href="bolt_configure_orchestrator.md" format="markdown"/>
         <topicref href="bolt_connect_puppetdb.md" format="markdown"/>
+        <topicref href="hiera.md" format="markdown"/>
```

Comment:
The smallest nit: I think this is more thematically similar to applying manifest blocks, so should go under `apply_manifest_blocks.md` in the nav.